test/e2e/auth/node_authz.go

```diff
@@ -179,4 +181,18 @@ var _ = SIGDescribe("[Feature:NodeAuthorizer]", func() {
 		err := c.CoreV1().Nodes().Delete("foo", &metav1.DeleteOptions{})
 		Expect(apierrors.IsForbidden(err)).Should(Equal(true))
 	})
+
+	framework.ConformanceIt("The kubelet's main port 10250 should fail with the Forbidden error", func() {
```

Comment:
Don't add conformance here please. You can add a comment though to maybe do it in the future though.
cc @tallclair